Documentation/Localization.ru_RU/Settings.cfg

```diff
@@ -0,0 +1,64 @@
+# More information about this file:
+# https://docs.typo3.org/m/typo3/docs-how-to-document/main/ru-ru/GeneralConventions/FileStructure.html#settings-cfg
+
+[general]
+
+project     = Getting Started
+version     = main (development)
+release     = main (development)
+copyright   = since 2012 by the TYPO3 contributors
+
+[html_theme_options]
+
+# "Edit on GitHub" button
+github_repository    = TYPO3-Documentation/TYPO3CMS-Tutorial-GettingStarted
+github_branch        = main
+
+# Footer links
+project_home         = https://docs.typo3.org/m/typo3/tutorial-getting-started/main/ru-ru/
+project_contact      = https://typo3.slack.com/archives/C028JEPJL
+project_repository   = https://github.com/TYPO3-Documentation/TYPO3CMS-Tutorial-GettingStarted
+project_issues       = https://github.com/TYPO3-Documentation/TYPO3CMS-Tutorial-GettingStarted/issues
+project_discussions  =
+
+use_opensearch       =
+
+[intersphinx_mapping]
+
+
+t3start11        = https://docs.typo3.org/m/typo3/tutorial-getting-started/11.5/ru-ru/
+
+# Official TYPO3 manuals
+# h2document     = https://docs.typo3.org/m/typo3/docs-how-to-document/main/ru-ru/
+# t3cheatsheets  = https://docs.typo3.org/m/typo3/docs-cheatsheets/main/ru-ru/
+# t3contribute   = https://docs.typo3.org/m/typo3/guide-contributionworkflow/main/ru-ru/
+t3coreapi      = https://docs.typo3.org/m/typo3/reference-coreapi/main/ru-ru/
```

Comment:
Linking to the Core API in russian will not work as we do not have a russian translation and as you agreed never will have. I would suggest to link to english instead